### DISCUSSION TITLE
Outer white ring of slider heads in "Argon" skin look weird

### DISCUSSION THREAD

**Discussion by frenzibyte** [ppy/osu#24881] [Q&A]
Originally mentioned in [discord](https://discord.com/channels/188630481301012481/188630652340404224/1154457651523096628).

This has been bugging me for a while, but the gap between the outer white ring of the slider head circle and the underlying circle piece makes it look overall sort of disoriented.

![image](https://github.com/ppy/osu/assets/22781491/43a88362-9e6b-46d0-b43c-73c403a38d2c)

It looks especially off at this part:

![image](https://github.com/ppy/osu/assets/22781491/2d1a1898-b086-4204-8ce3-9595e0cd2842)

**Comment by peppy** [comment]
This is part of the skin's style. Personally I like it.

I'm not sure what a team cc is going to do here, so i'll remove it.
